### ISSUE TITLE
Create ZIM of OWASP

### ISSUE THREAD

**Issue by kelson42** [openzim/zim-requests#47]
_From @kelson42 on April 2, 2018 17:1_

Hi! Can OWASP have a zim? They use mediawiki. 

https://www.owasp.org/index.php/Main_Page
https://owasp.org/

_Copied from original issue: openzim/mwoffliner#267_

**Comment by RavanJAltaie** [comment]
Requested
https://farm.openzim.org/recipes/owasp

**Comment by RavanJAltaie** [comment]
Succeeded.

**Comment by Popolechien** [comment]
Only first page was captured

**Comment by benoit74** [comment]
OWASP is not a Mediawiki anymore.
Website seems pretty big, and seems to have significant "outside" resources, many subdomains.
It is very unclear what we want to ZIM from my PoV.

**Comment by Popolechien** [comment]
I think https://cheatsheetseries.owasp.org/ would be the most valuable low-hanging fruit.

**Comment by benoit74** [comment]
Indeed, the cheat sheet series provide valuable information, a short summary of everything OWASP would like to spread as knowledge and seems realistically feasible with Zimit.

Proposed metadata:

- Website URL: https://cheatsheetseries.owasp.org/ 
- License: [Creative Commons Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/).
- Desired ZIM Title: OWASP Cheat Sheet Series
- Desired ZIM Description: A concise collection of high value information on application security topics
- Desired ZIM Long Description: The OWASP Cheat Sheet Series was created to provide a concise collection of high value information on specific application security topics. These cheat sheets were created by various application security professionals who have expertise in specific topics.
- Desired ZIM Icon –png (URL or attach one): Let Zimit use website favicon, should be fine
- Language (ISO 639-3): eng
- Is this a MediaWiki?: no
- Are you the owner/admin of the webiste (in case we need IP whitelisting)?: no

**Comment by benoit74** [comment]
And recipe / ZIM name should be `owasp.org_en_cheatsheetseries`, so that we will group all OWASP ZIMs -should we create more - under the `owasp.org` project, with specific selections like `cheatsheetseries`
